lib/src/interfaces/Options.ts

```diff
@@ -5,16 +5,6 @@ import { ImageRequireSource, Insets } from 'react-native';
 // Only assign PlatformColor or DynamicColorIOS as a Color symbol!
 declare type Color = string | symbol;
 type FontFamily = string;
-type FontWeight =
```

Comment:
What's the motivation for removing `FontWeight`? Seems like it's more explicit and provides better auto completion.

Reply:
It's different from one font to another. https://developer.apple.com/fonts/system-fonts/

Reply:
Seems like there's a general set of weights and each font can support a different subset. If a font doesn't support a given `fontWeight` then the utility functions in RCTFont use the closest weight.

Since specifying a weight which isn't supported by font doesn't error (instead an approximation is used, which is fine) I feel we should keep the type and have it include all weights supported in [RCTFont](https://github.com/facebook/react-native/blob/d0871d0a9a373e1d3ac35da46c85c0d0e793116d/React/Views/RCTFont.mm#L40). WDYT?
The auto completion is really beneficial in these cases 🙏